content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md

```diff
@@ -0,0 +1,71 @@
+
+
+## [Shiming Zhang](https://github.com/wzshiming)
+
+Shiming Zhang is a Software Engineer with DaoCloud in Shanghai, China. 
+
+He has mostly been involved with `SIG Node` as a reviewer.
+
+As of his prior experience in this project, Shiming has been involved with Kubernetes at his company. His major contributions have mainly been bug fixes and feature improvements in an ongoing [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2712-pod-priority-based-graceful-node-shutdown), all revolving around SIG Node.
+
+Some of his major PRs are [fixing watchForLockfileContention memory leak](https://github.com/kubernetes/kubernetes/pull/100326), [fixing startupProbe behaviour](https://github.com/kubernetes/kubernetes/pull/101093), [adding Field status.hostIPs for Pod](https://github.com/kubernetes/enhancements/pull/2661).
+
+
+## [Paco Xu](https://github.com/pacoxu)
+
+Paco Xu works at DaoCloud, a Shanghai-based cloud-native firm. He works with the infra and the open source team, focusing on enterprise cloud native platforms based on Kubernetes.
+
+He started with Kubernetes in early 2017 and his first contribution was in March 2018. He started with a bug that he found, but his solution was not that graceful, hence wasn't accepted. He then started with some good first issues, which helped him to a great extent. In addition to this, from 2016 to 2017, he made some minor contributions to Docker.
+
+Currently, Paco is a reviewer for `kubeadm` and `SIG Node`. He states that the maintainers have helped him a lot in learning and contributing to the project. According to him, the best thing one can do is to learn how the owners work on the project.
+
+Paco also emphasizes on contributing to an open source project if one is using it. For him, an open source project is like a book to learn. One can learn a lot from an open source project and get inspirisions during disucussions with the project maintainers.
+
+
+## [Jintao Zhang](https://github.com/tao12345666333)
+
+Jintao Zhang is presently employed at API7, where he focuses on ingress and service mesh.
+
+In 2017, he encountered an issue which led to a community discussion and his contributions to Kubernetes started. Before contributing to Kubernetes, Jintao was a long-time contributor to Docker-related open source projects. 
+
+Currently Jintao is a reviewer for the `Ingress-nginx` project.
+
+He suggests to follow open source commercial companies, if one wants to do open source full-time. For new contributors Jintao says that if anyone wants to make a significant contribution to an open source project, then they should choose the project based on their interests and should generously invest time.
```

Comment:
Maybe following them on socials and trying to understand their projects and stuff. I don't really know what this means.

Reply:
Need to figure that out, then, before we can publish.

Reply:
This still needs to either be fixed, or cut.  Thanks!

Reply:
@jberkus Do you have any suggestions for this as well?

Reply:
No, because I have no idea what was meant by it.  Did you query Jintao?

Reply:
Hey, this feedback is still unresolved, which is why we have not published.

Reply:
@jberkus I don't know how but my comment got deleted which consisted of Jintao's response to this.

So I pinged him regarding this to provide more context to his statement. He says
`At present, there are many open source commercialization companies.
For example I am working on [API7.ai](http://api7.ai/), of course there are many well known companies like HashiCorp, MongoDB, Docker Inc. etc. In these companies, full-time open source positions are often offered. Engineers can focus on open source projects and communities. And for these companies, open source projects and communities are the cornerstone of their success. So they will be more willing to invest in this`

How should we accommodate this into the blog?

Reply:
```suggestion
He suggests keeping track of job opportunities at open source companies so that you can get a job contributing full time. For new contributors Jintao says that if anyone wants to make a significant contribution to an open source project, then they should choose the project based on their interests and should generously invest time.
```